src/lib/runtime_config/runtime_config.ml

```diff
@@ -478,42 +475,6 @@ module Json_layout = struct
       ; slot_chain_end : int option [@default None]
       ; minimum_user_command_fee : Currency.Fee.t option [@default None]
       ; network_id : string option [@default None]
-      ; client_port : int option [@default None] [@key "client-port"]
```

Comment:
*Question*

This type has json derivers, so i think changing this affects graphql behavior for the "runtimeConfig" field. Is submitting a broken change ok here? It is annoying, but perhaps we need a versioned type.